internal/services/cdn/cdn_frontdoor_firewall_policy_resource.go

```diff
@@ -417,6 +418,7 @@ func resourceCdnFrontDoorFirewallPolicy() *pluginsdk.Resource {
 														string(frontdoor.ActionTypeLog),
 														string(frontdoor.ActionTypeBlock),
 														string(frontdoor.ActionTypeRedirect),
+														"AnomalyScoring", // Only valid with 2.0 and above
```

Comment:
We should make sure this exists in swagger & if not open an issue to ensure it is added in future SDKs

Reply:
Opened [issue](https://github.com/Azure/azure-rest-api-specs/issues/21435)